src/clients/SnapshotApi.ts

```diff
@@ -0,0 +1,174 @@
+import { Web3Provider } from '@ethersproject/providers'
+import { Wallet } from '@ethersproject/wallet'
+import snapshot from '@snapshot-labs/snapshot.js'
+import Client from '@snapshot-labs/snapshot.js/dist/sign'
+import { CancelProposal, Proposal, ProposalType } from '@snapshot-labs/snapshot.js/dist/sign/types'
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
+import env from 'decentraland-gatsby/dist/utils/env'
+
+import { ProposalInCreation, ProposalLifespan } from '../services/ProposalService'
+import { SNAPSHOT_ADDRESS } from '../entities/Snapshot/constants'
+import { getEnvironmentChainId } from '../modules/votes/utils'
+
+import { SnapshotGraphql } from './SnapshotGraphql'
+import { trimLastForwardSlash } from './utils'
+
+const SNAPSHOT_PROPOSAL_TYPE: ProposalType = 'single-choice' // Each voter may select only one choice
```

Comment:
This is something we can document in the README. I don't remember if we have a "Snapshot" config section but we could add it (later).